website/docs/reference/resource-configs/spark-configs.md

```diff
@@ -28,12 +28,12 @@ When materializing a model as `table`, you may include several optional configs
 
 </Changelog>
 
-dbt seeks to offer useful, intuitive modeling abstractions by means of its built-in configurations and <Term id="materialization">materializations</Term>. Because there is so much variance between Apache Spark clusters out in the world—not to mention the powerful features offered to Databricks users by the Delta file format and custom runtime—making sense of all the available options is an undertaking in its own right.
+dbt seeks to offer useful, intuitive modeling abstractions by means of its built-in configurations and <Term id="materialization">materializations</Term>. Because there is so much variance between Apache Spark clusters out in the world—not to mention the powerful features offered to Databricks users by the Delta file format and custom runtime or Hudi file format with Apache Spark runtime —making sense of all the available options is an undertaking in its own right.
```

Comment:
i feel this sentence is now trying to accomplish to many things. I understand why you want to mention Hudi alongside Delta, but maybe we can break it into two to three standalone sentences?

Reply:
Yep, good point, updated into two sentences.